pom.xml

```diff
@@ -76,7 +75,33 @@
     <dependency>
       <groupId>life.qbic</groupId>
       <artifactId>data-model-lib</artifactId>
-      <version>1.8.0</version>
+      <version>1.9.0-SNAPSHOT</version>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+            <configuration>
+              <archive>
+                <manifest>
+                  <mainClass>life.qbic.cli.PostregistrationToolEntryPoint</mainClass>
+                </manifest>
+              </archive>
+              <descriptorRefs>
+                <descriptorRef>jar-with-dependencies</descriptorRef>
+              </descriptorRefs>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
What does this do? Why is it needed now and not before?

Reply:
This was in the pom of the current development branch(1.5.0-SNAPSHOT). These are good questions, I'll check out why and when it was added .

Reply:
This should not have been removed! The assembly of all dependencies into an full artifact is crucial for the deployment into i.e. the openBIS environment. That's why this plugin was included.